generator.go

```diff
@@ -65,12 +65,16 @@ func (d *generator) Generate(opts *GenerateOptions) error {
 		if err := rows.Scan(&table); err != nil {
 			return err
 		}
+		exclude := false
 		for _, e := range opts.Exclude {
 			if e == table {
+				exclude = true
```

Comment:
How about jumping after `append()` by `goto`?

Reply:
I avoided using goto, but it looks good.
I fixed in d8b387d.
Label EOL stands for End of Loop.